server/commands.lua

```diff
@@ -253,6 +253,64 @@ lib.addCommand('setjob', {
     end
 end)
 
+
+--- ADMIN COMMAND
+
+
+lib.addCommand('changejob', {
+    help = locale("command.changejob.help"),
+    params = {
+        { name = locale("command.changejob.params.id.name"), help = locale("command.changejob.params.id.help"), type = 'playerId' },
+        { name = locale("command.changejob.params.job.name"), help = locale("command.changejob.params.job.help"), type = 'string' },
+    },
+    restricted = 'group.admin'
+}, function(source, args)
+    local player = GetPlayer(tonumber(args[locale("command.changejob.params.id.name")]) --[[@as number]])
+    if not player then
+        Notify(source, locale('error.not_online'), 'error')
+        return
+    end
+
+    SetPlayerPrimaryJob(player.PlayerData.citizenid, tostring(args[locale("command.changejob.params.job.name")]))
+end)
+
+lib.addCommand('addjob', {
+    help = locale("command.addjob.help"),
+    params = {
+        { name = locale("command.addjob.params.id.name"), help = locale("command.addjob.params.id.help"), type = 'playerId' },
+        { name = locale("command.addjob.params.job.name"), help = locale("command.addjob.params.job.help"), type = 'string' },
+        { name = locale("command.addjob.params.grade.name"), help = locale("command.addjob.params.grade.help"), type = 'number', optional = true}
+    },
+    restricted = 'group.admin'
+}, function(source, args)
+    local player = GetPlayer(tonumber(args[locale("command.addjob.params.id.name")]) --[[@as number]])
+    if not player then
+        Notify(source, locale('error.not_online'), 'error')
+        return
+    end
+    if args[locale("command.addjob.params.grade.name")] then
```

Comment:
It might be clearer to store the arg into a variable and dereferencing it one time. Then you can also make a default value of 0, only calling AddPlayerToJob once.